src/app/MessageProvider.tsx

```diff
@@ -18,17 +22,23 @@ async function polyfill(locale: 'en') {
 
 polyfill('en');
```

Comment:
@TomWoodward should this now be called from within the component with the appropriate locale?

Reply:
yea, and probably use state to wait until it finishes before you render